packages/web/src/scss/components/Icon/index.html

```diff
@@ -0,0 +1,80 @@
+{{#> layout }}
+  <h2 class="typography-heading-medium-text mt-1100 mb-800">Icon</h2>
+  <h3 class="typography-heading-small-text mt-1100 mb-800">Design only</h3>
+  <svg class="Icon" width="24" height="24">
```

Comment:
👀 
![obrazek](https://user-images.githubusercontent.com/5614085/184175966-817240a8-019a-426b-ad5e-c36160b5bd8c.png)

Reply:
It works for me, but I sent an update. Can you try it again? Thanks. (btw `yarn (install)` is required and then `start` or `build`)